src/test/system_tests/ceph_s3_tests/test_ceph_s3_constants.js

```diff
@@ -1,40 +1,58 @@
 /* Copyright (C) 2022 NooBaa */
 "use strict";
 
+const cephalt_name = 'cephalt';
+const cephtenant_name = 'cephtenant';
+
+// NC ceph tests paths for bucket creation
+const FS_ROOT_1 = '/tmp/nsfs_root1/';
+const FS_ROOT_2 = '/tmp/nsfs_root2/';
+
 const CEPH_TEST = {
     test_dir: 'src/test/system_tests/ceph_s3_tests/',
     s3_test_dir: 's3-tests/',
     ceph_config: 'test_ceph_s3_config.conf',
     tox_config: 'tox.ini',
     new_account_params: {
-        name: 'cephalt',
+        name: cephalt_name,
         email: 'ceph.alt@noobaa.com',
         has_login: false,
         s3_access: true,
     },
     new_account_params_tenant: {
-        name: 'cephtenant',
+        name: cephtenant_name,
         email: 'ceph.tenant@noobaa.com',
         has_login: false,
         s3_access: true,
     },
+    nc_cephalt_account_params: {
+        name: cephalt_name,
+        uid: 1000,
+        gid: 1000,
+        new_buckets_path: FS_ROOT_1
+    },
+    nc_cephtenant_account_params: {
+        name: cephtenant_name,
+        uid: 2000,
+        gid: 2000,
+        new_buckets_path: FS_ROOT_2
+    },
+    nc_anonymous_account_params: {
+        anonymous: true,
+        uid: process.getuid(),
+        gid: process.getgid()
```

Comment:
We can use constant uid and gid? (3000 for example)

Reply:
I kept it as it was, if you feel this needs to be changed, please add it to the small refactoring issue